configure.ac

```diff
@@ -1588,8 +1588,8 @@ AC_MSG_NOTICE([
   OpenThread NCP-MTD support                : ${enable_ncp_app_mtd}
   OpenThread NCP-FTD support                : ${enable_ncp_app_ftd}
   OpenThread NCP-BUS Configuration          : ${with_ncp_bus}
-  OpenThread NCP Vendor Hook Source         : ${with_ncp_vendor_hook_source}
-  OpenThread NCP Spinel Encrypter           : ${with_ncp_spinel_encrypter_libs}
+  OpenThread NCP Vendor Hook Source         : ${with_ncp_vendor_hook_source-none}
```

Comment:
better to set these values in `AC_ARG_WITH` (i.e. in the `action-if-not-present`)?